sparse-core/examples/vector.rs

```diff
@@ -0,0 +1,22 @@
+//! Example file for creating vectors.
+
+use sparse_core::local::indexable_space::LocalIndexableVectorSpace;
+use sparse_traits::Element;
+use sparse_traits::IndexableVector;
+use sparse_traits::LinearSpace;
+use sparse_traits::Norm2;
+
+fn main() {
+    let space = LocalIndexableVectorSpace::<f64>::new(10);
+    let mut vec = space.create_element();
```

Comment:
Would this naming be acceptable? I feel like we'll be eternally swimming against the current if we choose `element`.
```suggestion
    let mut vec = space.create_vector();
```

Reply:
I am torn about this. Most users will only want to deal with vectors. So then naming it ```create_vector``` makes sense. But it goes against the spirit of having a library that works for arbitrary types of spaces.

Reply:
What are you thinking of? "Vector space" and "linear space" are synonyms for the abstract mathematical objects (at least as I've seen them) and I'm comfortable with "vectors" meaning that (with more specific names for elements of $\symbf R^n$).